ADOPTERS.md

```diff
@@ -0,0 +1,7 @@
+# Athenz adopters
+
+- [Yahoo](http://www.yahoo.com)
```

Comment:
Even though it would redirect, should we change the url to https?

Reply:
yes, I just fixed it.